tests/test-sources/plugins/none-ls.nix

```
@@ -98,7 +98,7 @@
           cbfmt.enable = !pkgs.stdenv.isDarwin;
           eslint.enable = true;
           eslint_d.enable = true;
-          fantomas.enable = pkgs.stdenv.isLinux;
+          fantomas.enable = pkgs.stdenv.isLinux || pkgs.stdenv.isDarwin;
```

```suggestion
          fantomas.enable = true;
```
This should work I guess.